packages/universal-components/src/RangeDatePicker/DayItemArrow/DayItemArrow.android.js

```diff
@@ -0,0 +1,77 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+
+import { StyleSheet } from '../../PlatformStyleSheet';
+import { ExtendedTouchable } from '../../ExtendedTouchable';
+import { type Props } from './DayItemArrowTypes';
+
+export default class DayItemArrow extends React.Component<Props> {
+  renderArrow = () => (
+    <View style={styles.touchableContainer}>
+      <View
+        style={[
+          styles.arrow,
+          this.props.direction === 'left'
+            ? styles.arrowLeft
+            : styles.arrowRight,
+        ]}
+      />
+    </View>
+  );
```

Comment:
Id rather see that you create another component.

Reply:
Done